en/ros/README.md

```diff
@@ -1,45 +1,46 @@
-# Robotics using ROS
+# Interfacing with PX4 using ROS
 
-[ROS](http://www.ros.org/) (Robot Operating System) is a general purpose robotics library that can be used with PX4 for [offboard control](../ros/mavros_offboard.md).
-It uses the [MAVROS](../ros/mavros_installation.md) node to communicate with PX4 running on hardware or using the [Gazebo Simulator](../simulation/ros_interface.md).
+[ROS](http://www.ros.org/) (Robot Operating System) is a general purpose robotics library that can be used with the PX4 Autopilot for powerful applications. There are a couple of ways for ROS developers to interface with PX4, that will be mainly defined by your hardware and software architecture.
 
-This section contains topics about using ROS for offboard control with PX4.
+The PX4 Autopilot supports both ROS 1 and ROS 2, through its two main interfaces: [MAVLink](../middleware/mavlink.md) and [uORB](../middleware/uORB.md). While both interfaces have a solid foundation, they have different goals, and it will be up to the developer to choose which one best fits their application architecture.
```

Comment:
Just call it "PX4" after the first instance (i.e no need to spell out The PX4 Autopilot)